config/sampleCfg_Legacy2018.cfg

```diff
@@ -1,149 +1,168 @@
+WWToLNuQQ    = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_WWToLNuQQ
+
+WZTo1L1Nu2Q  = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_WZTo1L1Nu2Q
+WZTo1L3Nu    = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_WZTo1L3Nu
+WZTo2L2Q     = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_WZTo2L2Q
+WZTo3LNu     = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_WZTo3LNu
+
+ZZTo2L2Nu    = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_ZZTo2L2Nu
+ZZTo2L2Q     = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020SKIM_ZZTo2L2Q
+ZZTo2Q2Nu    = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_ZZTo2Q2Nu
+ZZTo4L       = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_ZZTo4L
+
+ZH_HBB_ZLL             = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_ZH_HBB_ZLL
+#ZH_HBB_ZQQ             = /data_CMS/cms/amendola/HHLegacy_2018_SKIMS/SKIMS_22Jan2020/SKIM_ZH_HBB_ZQQ  # NOT PRODUCED FOR 2018
+ZH_HTauTau             = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_ZH_HTauTau
+ggHTauTau              = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_ggHTauTau
+VBFHTauTau             = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_VBFHTauTau
+WplusHTauTau           = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_WplusHTauTau
+WminusHTauTau          = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_WminusHTauTau
+ttHJetTononBB          = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_ttHJetTononBB
+ttHJetToBB             = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_ttHJetToBB
+
+ZZZ                    = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_ZZZ
+WWW                    = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_WWW
+WWZ                    = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_WWZ
+WZZ                    = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_WZZ
+
+TTWJetsToLNu           = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_TTWJetsToLNu
+TTWJetsToQQ            = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_TTWJetsToQQ
+TTWW                   = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_TTWW
+TTZToLLNuNu            = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_TTZToLLNuNu
+TTWZ                   = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_TTWZ
+TTZZ                   = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_TTZZ
 
 #########
 
-VBFSM=/data_CMS/cms/amendola/HH2017Skim_Jan2019/SKIMS_3Apr2019_Run2017_0jets/SKIM_VBFSM
-GGFSM=/data_CMS/cms/amendola/HH2017Skim_Jan2019/SKIMS_3Apr2019_Run2017_0jets/SKIM_GGHSM
+VBFSM    = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_VBFHH_CV_1_C2V_1_C3_1
+GGHHSM   = /data_CMS/cms/amendola/HHLegacy_2018_v2/SKIMS_8May2020/SKIM_GGHHSM
```

Comment:
underscore missing -> SKIM_GGHHSM -> SKIM_GGHH_SM
(sorry, I really wanted to put an underscore in that process name :) )